lib/components/Stepper/Stepper.docs.tsx

```diff
@@ -282,6 +282,31 @@ const docs: ComponentDocs = {
           </Stepper>,
         ),
     },
+    {
+      label: 'Alignment',
+      description: (
+        <>
+          <Text>
+            The Stepper is center-aligned by default, but this can be customised
+            via the align prop.
```

Comment:
Should `align` be emphasised here since it's a prop name?
```suggestion
            via the <Strong>align</Strong> prop.
```